dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -441,12 +443,13 @@ private static CharSequence generateCoverageReport(Set<Event> coveredEvents, Pro
         final BranchEquivalence cf = analysisContext.requires(BranchEquivalence.class);
 
         final Set<Event> programEvents = program.getEvents(MemEvent.class).stream()
-                .filter(e -> e.hasCLine() && e.hasOId()).collect(Collectors.toSet());
+                // TODO: Can we have events with source information but without parse id?
+                .filter(e -> e.hasMetadata(SourceLocation.class) && e.hasMetadata(ParseId.class))
+                .collect(Collectors.toSet());
         
         // Track (covered) events and branches via oId
-        final Set<Integer> branches = new HashSet<>();
-        final Set<Integer> coveredOIds = new HashSet<>();
```

Comment:
Were the `coveredOIds` really never used?

Reply:
Yes, they are unused in development.